product_docs/docs/epas/17/reference/oracle_compatibility_reference/epas_compat_sql/22_create_directory.mdx

```diff
@@ -22,7 +22,11 @@ CREATE DIRECTORY <name> AS '<pathname>'
 
 ## Description
 
-The `CREATE DIRECTORY` command creates an alias for a file system directory pathname. You must be a database superuser to use this command.
+To create directories, you must have the `CREATE ANY DIRECTORY` system privilege.
+
+When you create a directory, you are automatically granted the `READ`, `WRITE`, and `EXECUTE` privileges on the directory, and you can grant `READ`, `WRITE`, and `EXECUTE` privileges to other users and roles. The superuser can also grant these privileges to other users and roles.
```

Comment:
There are only Read and Write privileges on directory. There is no execute privilege

Reply:
Removed `EXECUTE'